Userland/Libraries/LibELF/Arch/aarch64/tls.S

```diff
@@ -60,4 +60,6 @@
 .type __tlsdesc_static,@function
 __tlsdesc_static:
     ldr x0, [x0, #8]
+    // The first static TLS block is 16 bytes after the thread pointer on AArch64.
```

Comment:
typo in commit message for ` LibELF+LibC: Add support for Variant I of the TLS data structures `

> We currently only supported Variant II which is used by x86-64.
Variant II is used by both AArch64 (when using the traditional
non-TLSDESC model) and RISC-V, although with small differences.

Should say "Variant I is used by both AArch64 ...", right?